allen-cell-animated/simularium-website
# Fix/skip frame bugs (temp fix) (Update viewer to v2.9.6)

Problem
=======
See https://github.com/allen-cell-animated/simularium-viewer/pull/131

Solution
========
Upgraded simularium-viewer to 2.9.6 which contains a temporary fix for the bugs described in the above PR.

Steps to Verify:
----------------
1. Pull this branch and `npm start`
2. Verify that the bugs described in the above PR are no longer reproducible
